runtime/compiler/x/amd64/codegen/AMD64JNILinkage.cpp

```diff
@@ -1181,14 +1181,25 @@ void TR::AMD64JNILinkage::cleanupReturnValue(
       // type so that we sign and zero extend the narrower integer return types properly.
       //
       TR_X86OpCodes op;
+      TR::SymbolReference      *callSymRef = callNode->getSymbolReference();
       TR::ResolvedMethodSymbol *callSymbol = callNode->getSymbol()->castToResolvedMethodSymbol();
       TR_ResolvedMethod *resolvedMethod = callSymbol->getResolvedMethod();
 
       bool isUnsigned = resolvedMethod->returnTypeIsUnsigned();
+bool isBool;
```

Comment:
Left behind unused declaration from debugging

Reply:
Fixed in commit https://github.com/eclipse/openj9/pull/4086/commits/6758341778bba8d756cea2bfa472e9fa49d3165b